app/views/my_modules/_recent_protocol_dropdown.html.erb

```diff
@@ -0,0 +1,22 @@
+<% display_status = protocol.description.empty? && protocol.steps.count.zero? %>
+<% if Protocol.recent_protocols(current_user, current_team, 14).count.positive? %>
+  <div class="my-module-recent-protocols"
+    style="display: <%= display_status ? '' : 'none' %>"
+    data-update-url="<%= load_from_repository_protocol_path(protocol) %>"
+  >
+    <div class="btn-group">
+      <div class="title">Recent protocols from the Repository</div>
+      <div
+        class="dropdown-button"
+        title="Recent protocols"
+        data-toggle="dropdown"
+        aria-haspopup="true"
+        aria-expanded="false">
+        <span class="caret"></span>
+        <span class="sr-only">Toggle Dropdown</span>
```

Comment:
Same here

Reply:
I don't know for what this text, it is in every dropdown and without I18n. Can't find cases how to see this text.